src/app/CommandSender.cpp

```diff
@@ -82,6 +82,29 @@ CHIP_ERROR CommandSender::SendCommandRequest(const SessionHandle & session, Syst
     return SendInvokeRequest();
 }
 
+CHIP_ERROR CommandSender::SendGroupCommandRequest(const SessionHandle & session, System::Clock::Timeout timeout)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    VerifyOrExit(mState == State::AddedCommand, err = CHIP_ERROR_INCORRECT_STATE);
+
+    err = Finalize(mPendingInvokeData);
+    SuccessOrExit(err);
+
+    // Create a new exchange context.
+    mpExchangeCtx = mpExchangeMgr->NewContext(session, this);
+    VerifyOrExit(mpExchangeCtx != nullptr, err = CHIP_ERROR_NO_MEMORY);
+    mpExchangeCtx->SetResponseTimeout(timeout);
```

Comment:
This is meaningless for a groupcast, no?

Reply:
It is meaningless, removed it and changed the function signature.